Makefile

```diff
@@ -142,6 +147,17 @@ release-docker: docker
 	@docker push rtradetech/temporal:$(TEMPORALVERSION)
 	@echo "===================          done           ==================="
 
+# download and setup gvisor runtime
+.PHONY: gvisor
+gvisor:
+	wget https://storage.googleapis.com/gvisor/releases/nightly/latest/runsc -O /tmp/runsc
```

Comment:
`./tmp` is what i was thinking 😋general rule of thumb, try not to fiddle with root-level directories

Reply:
(since `./tmp` is in `gitignore`)

Reply:
hah good point :man_facepalming: